kustomize/overlays/sandbox/admindeployment/kustomization.yaml

```diff
@@ -16,6 +16,7 @@ configMapGenerator:
   files:
 # Files are read in alphabetical order. To ensure that we create the namespace first, prefix the file name with "aa".
     - clusterresource-templates/aa_namespace.yaml
+    - clusterresource-templates/ab_project-resource-quota.yaml
```

Comment:
nit: doesn't need the `aa_` we just do that so that namespaces get created first (golang reads the files in alphabetically)

Reply:
Hmm... when these are done as config maps then, what order are they run in?

Reply:
the config maps are mounted as files here, no?